prereise/gather/griddata/hifld/data_process/topology.py

```diff
@@ -311,7 +311,11 @@ def add_interconnects_by_connected_components(
     revised_g = nx.convert_matrix.from_pandas_edgelist(lines, "SUB_1_ID", "SUB_2_ID")
     sorted_interconnects = sorted(nx.connected_components(revised_g), key=len)[::-1]
     if len(sorted_interconnects) != len(interconnect_size_rank):
-        raise ValueError("Interconnections were not separated successfully")
+        interconnect_sizes = [len(c) for c in sorted_interconnects]
+        raise ValueError(
+            f"Interconnections were not separated successfully {interconnect_sizes}"
```

Comment:
Just realized we require python 3.8 now, so we can add the equals sign to print name=value, e.g. 

```
In [1]: x = [1,2]

In [2]: f"wow so cool {x=}"
Out[2]: 'wow so cool x=[1, 2]'
```

Reply:
Wow so cool!

Reply:
Fancy!